packages/cubejs-client-ws-transport/src/index.ts

```diff
@@ -11,18 +24,48 @@ class WebSocketTransportResult {
   }
 }
 
+type WebSocketTransportOptions = {
+  authorization: string,
+  apiUrl: string,
+  hearBeatInterval?: number,
```

Comment:
I think this should be `heartBeatInterval?`

Reply:
@hassankhan nice catch! I guess we've had it like this since the very beginning. 

I think we should add the correct one but keep the `hearBeatInterval` option (and deprecate it) for backward compatibility

Reply:
Ahh good point, I'd forgotten it had always been like that. Totally agree.

Reply:
Done ✅